packages/suite/src/components/suite/layouts/SuiteLayout/PageHeader/HeaderActionButton.tsx

```diff
@@ -13,14 +13,16 @@ export const HeaderActionButton = ({
     children,
 }: Pick<ButtonProps, 'onClick' | 'data-testid' | 'variant' | 'size' | 'isDisabled' | 'children'> &
     Pick<IconButtonProps, 'icon'>) => {
-    const layoutSize = useSelector(selectWindowSize);
-
-    const isMobileLayout = layoutSize === 'TINY';
     const commonProps = { icon, onClick, 'data-testid': dataTestId, variant, size, isDisabled };
 
-    return isMobileLayout ? (
-        <IconButton {...commonProps} />
-    ) : (
-        <Button {...commonProps}>{children}</Button>
+    return (
+        <>
+            <ConditionalRender container="content" maxWidth={breakpointThresholds.sm}>
```

Comment:
I must say, that I don't like this API. 

Better IMHO:
```ts
    <BreakPointContentSM>
        <Button {...commonProps}>{children}</Button>
    </BreakPointContentSM>
``` looks better to me.

But Adam has a strong opionion so I am gladly backing.